upi/openstack/021_network.yaml

```diff
@@ -22,12 +64,17 @@
       cidr: "{{ aci_cni['network_interfaces']['opflex']['subnet'] }}"
       allocation_pool_start: "{{ aci_cni['network_interfaces']['opflex']['subnet'] | next_nth_usable(10) }}"
       allocation_pool_end: "{{ aci_cni['network_interfaces']['opflex']['subnet'] | ipaddr('last_usable') }}"
-    when: os_networking_type == "CiscoACI"
+    when:
+      - os_networking_type == "CiscoACI"
+      - (containers_subnet.stdout == "") or
+        (aci_cni['network_interfaces']['opflex']['subnet'] not in containers_subnet_cidr.stdout_lines|list)
```

Comment:
same here for duplicate names of the subnet as it is "or" condition

Reply:
Addressed.